crud-server/src/routes.test.js

```diff
@@ -126,89 +126,703 @@ describe('Praas REST API', () => {
+            expect(Object.keys(res.body.errors)).to.include('suri');
+          });
+
+          it('should not allow non-url suri', async () => {
+            const ct = await helpers.fakeConduit();
+            ct.suri = 'not-in-url-format';
+            const res = await Api()
+              .post(`/conduits`)
+              .set('Authorization', `Token ${jakeUser.token}`)
+              .send({ conduit: ct });
+            expect(res.status).to.equal(422);
+            expect(res.body.error.errors[0].path).to.equal('suri');
+          });
+        });
+
+        context('testing suriType field...', () => {
+          it('should not allow no suriType', async () => {
+            const ct = await helpers.fakeConduit();
+            delete ct.suriType;
+            const res = await Api()
+              .post(`/conduits`)
+              .set('Authorization', `Token ${jakeUser.token}`)
+              .send({ conduit: ct });
+            expect(res.status).to.equal(422);
+            expect(Object.keys(res.body.errors)).to.include('suriType');
+          });
+
+          it('should not allow undefined suriType', async () => {
+            const ct = await helpers.fakeConduit();
+            ct.suriType = undefined;
+            const res = await Api()
+              .post(`/conduits`)
+              .set('Authorization', `Token ${jakeUser.token}`)
+              .send({ conduit: ct });
+            expect(res.status).to.equal(422);
+            expect(Object.keys(res.body.errors)).to.include('suriType');
+          });
+
+          it('should not allow null suriType', async () => {
+            const ct = await helpers.fakeConduit();
+            ct.suriType = null;
+            const res = await Api()
+              .post(`/conduits`)
+              .set('Authorization', `Token ${jakeUser.token}`)
+              .send({ conduit: ct });
+            expect(res.status).to.equal(422);
+            expect(Object.keys(res.body.errors)).to.include('suriType');
+          });
+
+          it('should not allow empty suriType', async () => {
+            const ct = await helpers.fakeConduit();
+            ct.suriType = '';
+            const res = await Api()
+              .post(`/conduits`)
+              .set('Authorization', `Token ${jakeUser.token}`)
+              .send({ conduit: ct });
+            expect(res.status).to.equal(422);
+            expect(Object.keys(res.body.errors)).to.include('suriType');
+          });
+
+          it('should allow only \'Airtable\', \'Google sheets\', or \'Smart sheet\'', async () => {
+            const ct = await helpers.fakeConduit();
+            ct.suriType = 'random';
+            const res = await Api()
+              .post(`/conduits`)
+              .set('Authorization', `Token ${jakeUser.token}`)
+              .send({ conduit: ct });
+            expect(res.status).to.equal(422);
+            expect(res.body.error.errors[0].path).to.equal('suriType');
+          });
+        });
```

Comment:
Please change the test description to 'should reject unsupported service types'.
Also, please remove backslash to escape single quote. Replace either with double quotes or use back tick.

Reply:
I have to change the test as `it('should reject unsupported service types "Airtable", "Google sheets", or "Smart sheet"', async() => {`